docs/Documentation/run_RBM.md

```diff
@@ -106,3 +104,187 @@ All necessary files have been created at this point (see Appendix A for a descri
 
 where first **Salmon_0.5** refers to **Salmon_0.5_Network** ("**\_Network**" is appended by the model software) and second"**Salmon_0.5**" is the output file name (This refers to ***full_path/output_name*** . In this case, if no directory is addressed, the output file will be directly located in the current directory). "**Salmon_0.5.Spat**" is also created and is a file that cross-references element numbers in the output file "**Salmon_0.5.Temp**" to lat/long for
 post-processing.
+
```

Comment:
In step 7:
- It says "see Appendix A for a description of the output" - output of what?
- "Simply run the temperature model with command line files as follows:" --> "Run the temperature model RBM with command line as follows:"
- "and second"Salmon_0.5" is the output file basepath" - also, do not capitalize "This" later in the parenthesis. The explanation in the parenthesis is good - I think it's better to move this explanation to step 3, where this explanation is needed for the first time.
- Explicitly say something like "There will be two direct output files after running RBM - suffixes ".Spat" and ".Temp" will be appended to the specified basepath as the full paths of the two output files. (I see that you are trying to explain why the ".Spat" file is necessary and how to use it - I think you can answer these two questions in more detail, and put in the `Appendix A` section together with the file descriptions you already have there, and refer to Appendix A in this step section.